1and2/1and2/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _1and2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        { // PRess enter or type anything and the the code will appear.
+            string name = Console.ReadLine();
+
+            String TomName = "Tom Jefferson";
+             int TomAge = 19;
```

Comment:
hardcoded values. Please use Console.Readline to read user input. Use int.Parse, Float.Parse to convert user input to numbers where applicable.